cmp/dataview/DataViewModel.js

```diff
@@ -88,26 +85,26 @@ export class DataViewModel {
 
         // We only have a single visible column in our DataView grid and rely on ag-Grid for sorting,
         // initially and through updates via transactions. For this reason, we set the field of our
-        // single column to the desired sort field. (The field setting should not have any other
-        // meaningful impact on the grid, since we use a custom renderer and mark it as complex to
-        // ensure each item re-renders on any record change.)
-        let field = 'id';
-        if (sortBy.length === 1) {
-            let sorter = sortBy[0];
-            if (!(sorter instanceof GridSorter)) sorter = GridSorter.parse(sorter);
-            field = sorter.colId;
-        }
+        // single visible column to the desired sort field while other fields are hidden columns to allow
+        // for multiple sorters and groupings or using setGroupBy and setSortBy methods on the model itself.
 
-        const columns = [{
-            field,
+        const columns = store.fields.filter(field => {
+            return field !== sortBy;
+        }).map(field => {
+            return {
+                colId: field,
```

Comment:
Not sure if I need colIds here, as field seems to be sufficient identification (at least in the toolbox examples)